scripts/format.sh

```diff
@@ -13,11 +13,13 @@
 # This script will use clang-tidy and clang-format to format code.
 #
 # Usage:
-#    ./format.sh <path to directory to format>
+#    ./format.sh <paths to directories or files to format>
 # 
 # We require clang-format 17.0.4. Other versions may produce different styles!
 # ---------------------------------------------------------------------------------
 
-find $1 -iname '*.h' -o -iname '*.hpp' -o \
+paths="$@"
+
+find $paths -iname '*.h' -o -iname '*.hpp' -o \
```

Comment:
shellcheck suggests something like (I added the input check because I forgot when using the script)
```suggestion
if [ $# -lt 1 ]; then
    echo "ERROR: At least one path to format required"
    exit 1
fi

paths=( "$@" )

find "${paths[@]}" -iname '*.h' -o -iname '*.hpp' -o \
```